codesign.sh

```diff
@@ -1,3 +1,5 @@
+#!/bin/sh
```

Comment:
I would be careful with this, it should be fine on MacOS. But it is not in the POSIX standard to have `[ ]` testing but is in natively bash.